Example/ConsentViewController/ViewController.swift

```diff
@@ -10,84 +10,63 @@ import ConsentViewController
 class ViewController: UIViewController {
     var consentViewController: ConsentViewController!
 
-    private func buildConsentViewController(showPM: Bool) throws -> ConsentViewController {
-        let consentViewController = try ConsentViewController(
+    private func buildConsentViewController(showPM: Bool, addToView parentView: UIView) {
+        do {
+            let consentViewController = try ConsentViewController(
                 accountId: 22,
                 siteName: "mobile.demo",
                 stagingCampaign: false
             )
-        // optional, set custom targeting parameters supports Strings and Integers
-        consentViewController.setTargetingParam(key: "CMP", value: String(showPM))
-
-        // optional, sets debug level defaults to OFF
-        consentViewController.debugLevel = ConsentViewController.DebugLevel.OFF
-
-        consentViewController.willShowMessage = { cvc in print("the message will show") }
 
-        // optional, callback triggered when message choice is selected when called choice
-        // type will be available as Integer at cvc.choiceType
-        consentViewController.onMessageChoiceSelect = { cvc in
-            print("Choice type selected by user", cvc.choiceType as Any)
-        }
-
-        consentViewController.onErrorOccurred = { error in print(error) }
-
-        // optional, callback triggered when consent data is captured when called
-        // euconsent will be available as String at cLib.euconsent and under
-        // PreferenceManager.getDefaultSharedPreferences(activity).getString(EU_CONSENT_KEY, null);
-        // consentUUID will be available as String at cLib.consentUUID and under
-        // PreferenceManager.getDefaultSharedPreferences(activity).getString(CONSENT_UUID_KEY null);
-        consentViewController.onInteractionComplete = { cvc in
-            do {
-                print(
-                    "\n eu consent prop",
-                    cvc.euconsent as Any,
-                    "\n consent uuid prop",
-                    cvc.consentUUID as Any,
-                    "\n eu consent in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.EU_CONSENT_KEY) as Any,
-                    "\n consent uuid in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.CONSENT_UUID_KEY) as Any,
+            consentViewController.onMessageReady = { controller in
+                parentView.addSubview(controller.view)
+                controller.view.frame = controller.view.superview!.bounds
```

Comment:
Could we remove this force unwrap by accessing the `parentView.bounds` directly? Also, might make it a bit clearer that the `parentView` size is being adhered to!

```suggestion
                controller.view.frame = parentView.bounds
```

Reply:
Yep. Much easier.